cmd/datamon/cmd/bundle_mount.go

```diff
@@ -13,22 +19,94 @@ import (
 	"github.com/spf13/cobra"
 )
 
+func registerSIGINTHandlerMount(mountPoint string) {
```

Comment:
Should this code go some place else? Either a new file or a new package?

Reply:
this (like much of the daemonization impl) is lifted out of [gcsfuse](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/main.go), which suggests that `registerSIGINTHandlerMount` is be in the same package as much of the CLI.

i'm open to concrete suggestion about where this should go, of course, and will put it in a new file in case that's clearer.

tangentially, the distinction between physical layout and logical layout is an interesting dichotomy in go:  unlike C, the language has a notion of logical dependencies -- i.e. packages.  like C (albeit in a implementation-dependent way), physical layout in go has a natural correspondence with logical deps -- i.e. [intra-package initialization](https://golang.org/ref/spec#Package_initialization).

that said, the opinionated note i want to make (and the reason that i'm generally not too concerned about physical layout as opposed to logical deps), is that i'm not particularly concerned about physical layout in go, because the correspondence between it and logical deps is a fairly weak one compared to packages .. something that probably only makes much difference in terms of `init()` functions and the like.

documentation (and api design and a bunch of other ways to make the codebase more user-friendly) are, of course, super important.  let's try to maintain the "design stamina" on these and related issues.